torch/csrc/jit/runtime/operator.h

```diff
@@ -249,6 +249,8 @@ TORCH_API std::string canonicalSchemaString(const FunctionSchema& schema);
 TORCH_API const std::vector<std::shared_ptr<Operator>> getAllOperators();
 TORCH_API const std::vector<std::shared_ptr<Operator>>& getAllOperatorsFor(
     Symbol name);
+TORCH_API std::vector<std::shared_ptr<Operator>> getAllOperatorsForAtenFirst(
```

Comment:
nit: if this only called in python files, and only has one use, it's might be better to define it there instead of increasing libtorch's build size.